proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java

```diff
@@ -144,9 +144,12 @@ public void handle(KeepAlive alive) throws Exception
     @Override
     public void handle(Chat chat) throws Exception
     {
-        int maxLength = ( con.getPendingConnection().getVersion() >= ProtocolConstants.MINECRAFT_1_11 ) ? 256 : 100;
-        Preconditions.checkArgument( chat.getMessage().length() <= maxLength, "Chat message too long" ); // Mojang limit, check on updates
-
+        for(char character : chat.getMessage().toCharArray()) {
```

Comment:
String allows for random access, hence you don't have to convert it to a char array.

Reply:
You're right i will do it with charAt